lstchain/tests/test_lstchain.py

```diff
@@ -32,16 +34,20 @@
 
 
 def test_import_calib():
-    from lstchain import calib
+    pass
+
 
 def test_import_reco():
-    from lstchain import reco
+    pass
+
 
 def test_import_visualization():
-    from lstchain import visualization
+    pass
+
 
 def test_import_lstio():
-    from lstchain import io
+    pass
```

Comment:
Why the `pass` changes? These tests now do nothing.

Reply:
It's probably an unwanted change from your previous commits. I will try to revert them.

Reply:
I think it was the automatic refactor and optimizing imports options